### PR TITLE
Provision musictherapy.music.uiowa.edu on uiowa03

### PR DESCRIPTION
**Pull request by joewhitsitt**
ssl issue: https://github.com/uiowa/uiowa/issues/5161

### CODE REVIEW

**Review by cory-skeers — APPROVED** [review]

**Comment by joewhitsitt** [comment]
Sorry, I forgot to ask @mark-bennett-uiowa about whether musictherapy.music.uiowa.edu is okay to move forward with.

**Comment by mark-bennett-uiowa** [comment]
@joewhitsitt Well, it was the first I have heard of it. Music didnt contact me, and they didnt seem to have this site before. If this is a "lab" site, I dont seem to have any issues with it. I just need to make sure that music.uiowa.edu gets put on uiowa03 when their time to move comes up. You might as well add music.uiowa.edu and *.music.uiowa.edu to that SSL cert.

**Comment by mark-bennett-uiowa** [comment]
I am going to contact the only contact over there that I know is still around. The main contact for web left months ago.

**Comment by joewhitsitt** [comment]
Okay, let me know if I can merge this or if adjustments should be made.

**Comment by mark-bennett-uiowa** [comment]
Here is the response from Laura Goddard, referring to a grad student in music:

I can’t be sure what you’re referencing. 
Two things maybe related:

- I recently advised a graduate student to pursue creating a website through SiteNow.  He is a music therapy graduate student, and his research involves creating a platform to assess hearing among our student population.  He wanted to have a page devoted to describing the project, asking students to take surveys etc; since this isn’t something I’d be able to maintain and he wanted to be able to edit it, I sent him (on the advice of our marketing person) the SiteNow route. 
- I know our music therapy area head is chomping at the bit to get marketing materials created for her area.  (We have been without a SOM marketing person since July 2021.)  I don’t know if she or someone else reached out to OSC for a separate site, or if they did, their rationale for doing so.

 
Basically, this is also news to me.

Not sure why they asked for a *.music.uiowa.edu site, rather than a *.sites, but it is where we are. Does anyone else have any issues with this? @mjoneill

**Comment by mark-bennett-uiowa** [comment]
@joewhitsitt I responded to the above email from Laura with instructions that with this subdomain site, this "lab" site needs to adhere to the strictest CLAS and OSC policies as far as branding, accessibility, and content, especially with this being a subdomain of the music.uiowa.edu site. It reflects on the School of Music even more than a *.lab or *.sites site would.

**Comment by joewhitsitt** [comment]
>Not sure why they asked for a *.music.uiowa.edu site, rather than a *.sites, but it is where we are. Does anyone else have any issues with this?

It isn't too late to shift things around. Based on the response, this doesn't sound like the Music Theory program. Michael wasn't in stand up this morning to give thoughts. I can bring this up again Monday morning.

I can break sipe.org.uiowa.edu out into a separate PR if that needs to move sooner.

**Comment by mark-bennett-uiowa** [comment]
> > Not sure why they asked for a *.music.uiowa.edu site, rather than a *.sites, but it is where we are. Does anyone else have any issues with this?
> 
> It isn't too late to shift things around. Based on the response, this doesn't sound like the Music Theory program. Michael wasn't in stand up this morning to give thoughts. I can bring this up again Monday morning.
> 
> I can break sipe.org.uiowa.edu out into a separate PR if that needs to move sooner.

Sounds great. I have brought this up to both Laura Goddard and Kristan Hellige, who are both admins to SoM and the Division of Performing Arts areas. Neither of them know necessarily who this came from, nor want the SoM involved in this site from an editing or maintenance perspective. Kristan said she could find out more information early Monday if we can hold on this for now.

**Comment by mjoneill** [comment]
@mark-bennett-uiowa 

> Not sure why they asked for a *.music.uiowa.edu site

If they are part of the school of music, I think it makes a lot of sense to nest there.  If they'd rather be on a *.sites. site, that's an option they are welcome to use also.

My assumption is that there's not a technical reason to avoid building out under .music.uiowa.edu

**Comment by mjoneill** [comment]
> Neither of them know necessarily who this came from, nor want the SoM involved in this site from an editing or maintenance perspective. Kristan said she could find out more information early Monday if we can hold on this for now.

^^^ important context.  I didn't read that on my first pass.

**Comment by mjoneill** [comment]
Yeah. If the school of music doesn't want the affinity in domain names, then I'd say push them to another preapproved domain 

My invisible decision tree is: if someone wants to be under a college/school/third-level department domain, ask the college/school/department (assuming CLAS web team is agnostic).  If the college/school/etc doesn't want it, then go to preapproved.

**Comment by mark-bennett-uiowa** [comment]
> Yeah. If the school of music doesn't want the affinity in domain names, then I'd say push them to another preapproved domain
> 
> My invisible decision tree is: if someone wants to be under a college/school/third-level department domain, ask the college/school/department (assuming CLAS web team is agnostic). If the college/school/etc doesn't want it, then go to preapproved.

Yeah, I just wanted to make sure that the SoM was okay with this before "attaching" them to their domain, much like we do with the Physics sites in a way. Considering the administrative team over at SoM didnt seem to know much about this, it threw some red flags, and I wanted to get a little more information before continuing.

**Comment by bspeare** [comment]
Closing this and going to start over with `musictherapy.sites.uiowa.edu`
